styles/themes/charcoal.less

```diff
@@ -0,0 +1,226 @@
+ * but for the original three themes, it is.
+ *
+ * -- For themes with non-w/b background/text, choose
+ * @border-color-base to be the default border color, and then
+ * choose medium and low contrast relative to the page background.
+ * @border-color-base-highc has been added specifically for WC.
+ * Not needed in charcoal until page-interfaces get themed.
+ */
+
+@border-color-base: #565656;         // default border color
+@border-color-base-mediumc: #808080; // medium c page background
+@border-color-base-lowc: #404040;    // low c page background.
+
+/* Attention-getting structures --- these are the old names */
+
+@satisfied: darkgreen;     // background-color?
+@not-satisfied: darkred;   // background-color
+@alertbar-color: teal;     // needs sub?
+@random-rule-header: @table-cell-base-mediumc; // for current lack of
+                     // better color name. Cornsilk on white themes
+                     // may need to change this.
+
+@blurb-box: @navbar-background;
+@comments-background: @navbar-background;
+
+@inprog: #b34700;
+@done-current: darkgreen;
+@done-previous: #b3002d;
+
+@subscr-hold-selected: darkgreen;
+
+@enabled: darkgreen;
+@disabled: #454545;
+@pending: #e67e00;
+
+@highlight: #b34700;
+
+/* stars -- commented out uses default */
+@star-gold: #d4af37;
+@star-silver: #b3b3b3;
+// @star-bronze: ; // #cd7f32;
+
+/* Progress bar */
+@progress-high: #009900;   // green family;
+@progress-medium: #ff6600; // orange;
+@progress-low: #b3002d;    // red family;
+
+/*--------------------------------------------------------*/
+/* Available Project listings */
+/* The colors below will override the default colors listed in theme.less */
+
+@P1-even: @page-background;
+@P1-odd: @table-cell-base-mediumc;
+@P2-even: @page-background;
+@P2-odd: @table-cell-base-mediumc;
+@P3-even: @page-background;
+@P3-odd: @table-cell-base-mediumc;
+@F1-even: @page-background;
+@F1-odd: @table-cell-base-mediumc;
+@F2-even: @page-background;
+@F2-odd: @table-cell-base-mediumc;
+@PP-even:@page-background;
+@PP-odd: @table-cell-base-mediumc;
+@PPV-even: @page-background;
+@PPV-odd: @table-cell-base-mediumc;
+@SR-even: @page-background;
+@SR-odd: @table-cell-base-mediumc;
+
+/* News items */
+/* if a color is not customized for the theme and needs to
+   be, uncomment the line and specify the color */
+
+@news-border: #787878; // shade of grey
+@news-normal: #ababab; // shade of grey
+@news-announce-high: #df206c; // default maroon, shade of red
+// @news-announce-medium: // shade of orange
+@news-announce-low: #8080ff; // shade of purple
+@news-celebrate: #17cf17; // shade of green
+@news-maintain: #f53d3d; // shade of red
+
+/* Overrides: for now, a place to stash things that will
+   likely be sorted out later into more logical places */
+
+textarea {
+    color: @page-text;
+    background-color: @page-background;
+    .default-border;
+}
+
+a {
+    text-decoration: none;
+}
+
+#navbar {
+    .current-tab {
+        // border-bottom: none;
```

Comment:
If we don't want this, we should remove it.

Reply:
I must have needed that at some point, but as far as I could tell, none of that definition was having any effect. I also moved the removal of the underline for links up just under the main color definitions.